contracts/LifCrowdsale.sol

```diff
@@ -109,9 +119,16 @@ contract LifCrowdsale is Ownable, Pausable {
     rate2 = _rate2;
     privatePresaleRate = _privatePresaleRate;
     foundationWallet = _foundationWallet;
-    marketMaker = _marketMaker;
-    minCap = _minCap;
-    maxPresaleWei = _maxPresaleWei;
+  }
+
+  // Set how the rate wei per USD, necesary to calculate with more precision the
+  // maxCap on the presale and distribution of funds on finalize.
+  // This function can be called only 10 blocks before the publicPresaleStartBlock and startBlock
+  function setWeiPerUSD(uint256 _weiPerUSD) onlyOwner {
```

Comment:
As I said above, I think we should use 2 separate variables for the presale and for the TGE. It will be more clear for everyone.